std/algorithm/setops.d

```diff
@@ -949,7 +949,8 @@ See also: $(LREF multiwayMerge)
 auto multiwayUnion(alias less = "a < b", RangeOfRanges)(RangeOfRanges ror)
 {
     import std.algorithm.iteration : uniq;
-    return ror.multiwayMerge.uniq;
+    import std.functional : not;
+    return multiwayMerge!less(ror).uniq!(not!less);
```

Comment:
The pred use for `uniq` doesn't seem correct. `uniq`'s pred is supposed to be for equality

Reply:
It is correct here for current uniq implementation. And it is a big task to invent a uniq implementation that will not work here. The `not!less` solutions seems to be the best here